packages/story-editor/src/app/media/local/actions.js

```diff
@@ -91,42 +91,6 @@ export const removeAudioProcessing =
     });
   };
 
-export const setBaseColorProcessing =
```

Comment:
Why change this? Is it related to this PR?

Reply:
I noticed that base color and blurhash for a single recorded video/image were generated and uploaded up to 3 times. I‘ve tweaked the logic as part of this PR.
Now it only happens once, as expected.

Will clean up the code & update the PR description accordingly soon.

Reply:
Do we need to do the same for poster images?

Reply:
The poster image was only generated once, all looked good there. But will check again.